build.sbt

```diff
@@ -31,6 +24,14 @@ lazy val cassandra = project
     Dependencies.Cassandra
   )
 
+lazy val files = project
```

Comment:
isn't `files` vs `file` worse than the full name? (that's why I choose that over `files`)